space_commands/space_launches.time

```diff
@@ -0,0 +1 @@
+2021-02-27T17:59:57
```

Comment:
Maybe this file should be added to the .gitignore as well?

Reply:
Yeah. I'll have to make sure it doesn't crash when the file isn't present then